test/httpProblemDetail.test.ts

```diff
@@ -0,0 +1,103 @@
+import { HttpProblemDetail } from '../src';
+import Joi from '@hapi/joi';
+
```

Comment:
Is this test `chai`, `jest`, or what? I am asking because maybe there is some other way than `Joi`. I don't say that it is wrong, but it is strange for me ;)

Reply:
`tsdx` uses `jest`. I checked it and jest doesn't have support for checking complex object struxture